resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -61,6 +62,8 @@ private[spark] class ExecutorPodsAllocator(
     podAllocationDelay * 5,
     conf.get(KUBERNETES_ALLOCATION_EXECUTOR_TIMEOUT))
 
+  private val driverPodReadinessTimeout = 5
```

Comment:
Like KUBERNETES_ALLOCATION_EXECUTOR_TIMEOUT, we need an official configuration instead of the magic number here. Could you add it to `resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala`?